### ISSUE TITLE
Some cron schedules are incorrectly interpreted as seconds

### ISSUE THREAD

**Issue by elahrvivaz** [citusdata/pg_cron#239]
Some valid cron syntax, such as [9,19,29,39,49,59 * * * *](https://crontab.guru/#9,19,29,39,49,59_*_*_*_*) is now being interpreted as running every 9 seconds after the changes in https://github.com/citusdata/pg_cron/pull/235. To recreate, try:

```
create or replace procedure test_cron(sleep_time integer)
language plpgsql
as $$
begin
	perform pg_sleep(sleep_time);
end;
$$;

select cron.schedule('test_a', '9,19,29,39,49,59 * * * *', 'call test_cron(1);');
```

The job should run on every 9th minute, but instead it runs every 9 seconds.

**Comment by marcocitus** [comment]
Thanks, shot myself in the foot with sscanf. Did a quick patch for now.

Fixed by [37b5687](https://github.com/citusdata/pg_cron/commit/37b56877bd1f27eac1b76196879c673125185084)

**Comment by elahrvivaz** [comment]
Thanks so much for the extremely quick turn-around!
